charts/drpc-nodecore/values.yaml

```
@@ -35,6 +35,10 @@ configMap:
   name: drpc-nodecore-config
   subPath: nodecore.config.yaml
 
+# Set to false to remove the nginx sidecar and route traffic directly to dRPC (no auth, internal cluster use only).
+nginxSidecar:
+  enabled: true
```

The current naming exposes implementation details. Users care about enabling/disabling authentication, not about nginx specifically. Consider renaming to:
```suggestion
auth:
  enabled: true
```